net/http/authz/authorizer.go

```diff
@@ -16,19 +16,28 @@ import (
 
 var ErrNotAuthorized = errors.New("not authorized")
 
+var builtinGrants []*Grant // initialized in loopback_authz.go
+
 type Authorizer struct {
 	raftDB        *raft.Service
 	raftPrefix    string
 	policyByRoute map[string][]string
+	extraGrants   map[string][]*Grant
```

Comment:
Hmm, okay. (They're not all necessarily built-in, though. See next PR https://github.com/chain/chain/compare/lorefactor...self-authz.)